src/sbt-test/sbt-protoc/golang/build.sbt

```diff
@@ -0,0 +1,27 @@
+val protobufVersion = "3.3.1"
```

Comment:
In the interest of minimizing redundancy and test running time, can you add only a single unit test for all the built-in generators, instead of a unit test for each individual generator?